ansible_specdoc/objects.py

```diff
@@ -214,7 +214,7 @@ def ansible_doc(self) -> Dict[str, Any]:
             "description": self.description,
             "type": str(self.type),
             "returned": self.returned,
-            "sample": json.loads("".join(self.sample)),
+            "sample": [json.loads(sample) for sample in self.sample],
```

Comment:
I think this would be a breaking change for existing templates but the Ansible Galaxy generated documentation should handle it fine. IMO this should be fine as long as we communicate it as part of the release 🙂